charts/adot-exporter-for-eks-on-ec2/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: adot-exporter-for-eks-on-ec2
 description: A Helm chart for collecting metrics using ADOT Collector to send to AWS monitoring services.
 type: application
-version: 0.10.0
-appVersion: "0.24.1"
+version: 0.11.0
+appVersion: "0.25.0"
 kubeVersion: ">=1.19.0-0"
```

Comment:
Do we need to update this as well?

Reply:
Yes, the CD is triggered by an increment to `version`. I believe `appversion` is a best practice to reflect the version of the application inside the chart and is the standard here.

Reply:
I meant the `kubeVersion`.

Reply:
Unclear if a new semver range should be applied. Created a new [issue](https://github.com/aws-observability/aws-otel-helm-charts/issues/84) to address this.